lamindb_setup/_check_setup.py

```diff
@@ -70,14 +71,48 @@ def _get_current_instance_settings() -> InstanceSettings | None:
         return None
 
 
+def _check_in_modules(module: str, isettings: InstanceSettings | None = None) -> None:
+    not_in_instance_msg = f"'{module}' is missing from this instance."
+
+    if isettings is not None:
+        if module not in isettings.modules:
+            raise RuntimeError(not_in_instance_msg)
```

Comment:
Can we throw a `ModuleWasntConfigured` error and say: "Please go to your instance settings page and it under "schema modules" if you want to use it."

Reply:
And `ModuleWasntConfigured` should not show a traceback by inheriting from `SystemException`.

Reply:
Should `ModuleWasntConfigured` inherit from `RuntimeError`?

Reply:
done